pulumi/infra/observability_env_vars.py

```diff
@@ -32,12 +32,14 @@ def otel_config(
     lightstep_endpoint: str = "ingest.lightstep.com:443",
     # This is optional because pulumi.config.get_bool returns Optional[bool]
     lightstep_is_endpoint_insecure: bool | None = False,
+    trace_sampling_percentage: float | None = 100.0,
```

Comment:
Can this be restructured so there's only one default value for this configuration variable in the codebase?

I've had bad experiences in the past playing "default value whack-a-mole" trying to figure out exactly where a given value is coming from, because every place it was used had a fallback.